.github/workflows/grpc.yml

```diff
@@ -0,0 +1,52 @@
+name: grpc-testsuite
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+jobs:
+  grpc-testsuite-1_37_1:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: grpc
+    steps:
+      - uses: actions/checkout@v2
```

Comment:
I think this is maybe redundant, as ddtrace is being checked out in `ddtrace` below